game.css

```diff
@@ -25,57 +30,57 @@ button{
   flex-direction: column;
 }
 
-.data{
+.data {
   position: relative;
-  top:5%;
+  top: 5%;
   display: flex;
   width: 100%;
   align-items: center;
-  justify-content:center;
+  justify-content: center;
   flex-direction: row;
 }
 
-.controls{
+.controls {
   position: relative;
-  bottom:15%;
+  bottom: 15%;
   display: flex;
-  width:100%;
+  width: 100%;
   justify-content: space-around;
 }
 
-#counter{
+#counter {
   text-align: center;
-  flex-grow:2;
+  flex-grow: 2;
   font-size: 4rem;
 }
 
-#div-collect-counter{
+#div-collect-counter {
   text-align: center;
-  flex-grow:1;
+  flex-grow: 1;
   font-size: 1.5rem;
   color: #42f4a7;
 }
 
-#div-balloon-counter{
+#div-balloon-counter {
   text-align: center;
-  flex-grow:1;
+  flex-grow: 1;
   font-size: 1.5rem;
 }
 
-#canvas{
+#canvas {
```

Comment:
not sure you need this anymore since it's commented out

Reply:
@tyeung2017